docs/drafts/BEP-0001.md

```diff
@@ -2,9 +2,9 @@
 
 ## Abstract
 
-The purpose of this BEP is to add populationAbundance(TAX:<species_id>) or pop(TAX:<species_id>) to the BEL Language. 
```

Comment:
Might we change `TAX` to `TAXONOMY` to reflect identifiers.org?

Reply:
It's so long comparatively - the prefix is longer than the ID (I did check identifiers.org and they use the numeric IDs not names).  I'd prefer to use TAX, but I'll submit to the majority on this if Natalie agrees.

Reply:
@ncatlett agreed